bedrock/releasenotes/models.py

```diff
@@ -145,6 +145,7 @@ class Note(RNModel):
     sort_num = None
     created = None
     modified = None
+    progressive_rollout = False
```

Comment:
I didn't see the corresponding migration for this. Is this a pseudo-model or something?

Reply:
Yeah it's a wrapper for a JSON array of objects, basically